content/sensu-go/6.2/operations/deploy-sensu/generate-certificates.md

```diff
@@ -260,7 +259,7 @@ Before you move on, make sure you have copied the certificates and keys to each
 
 - [Copy the Certificate Authority (CA) root certificate file][11], `ca.pem`, to each agent and backend.
 - [Copy all backend PEM files][12] to their corresponding backend systems.
-- [Copy all agent PEM files][13].
+- [Copy all agent PEM files][13] to the agent.
```

Comment:
This contradicts the instructions in lines 233 through 242, which say all of these files should be saved in the `/etc/sensu/tls` directory on each *backend*. Don't you copy all the backend cert/key *and* agent cert/key files to that same directory? I'm missing something because I don't understand what copying these files to the agent means.